spec/components/provider_interface/interview_and_course_summary_component_spec.rb

```diff
@@ -24,6 +24,16 @@
     expect(component).to include(interview.location)
   end
 
+  it 'applies a hidden field to the change link' do
```

Comment:
If we are to test this our spec should indicate why this is important.

```suggestion
  it 'change link has a hidden field to improve accessibility for visually impaired users' do
```